src/cephadm/box/util.py

```diff
@@ -4,6 +4,16 @@
 import sys
 from typing import Any, Callable, Dict
 
+class Colors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKCYAN = '\033[96m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
```

Comment:
I want to be outraged at embedding terminal-specific sequences in code, vs using `ncurses` or `terminfo`. or `tput`.  But pragmatically I must admit that these days the number of Linux users with terminals that aren't an ANSI variant is vanishingly small.

Reply:
Tbh I've never done colors in python so I went for the quickets solution I could find out there. My priority is merging this and then help zmc creating osds.

Reply:
Ack, just having a geezer moment ;)